### ISSUE TITLE
Incorrect Swager Definition for eav-data-attribute-option-interface 

### ISSUE THREAD

**Issue by vetshopdeveloper** [magento/magento2#18525]
### Preconditions

1. 2.2.6

### Steps to reproduce

1. create a rest call to `CatalogProductAttributeRepositoryV1Api.CatalogProductAttributeRepositoryV1GetListGet` with search parameters of `attribute_set_id > 0` 

or 

any other endpoint that returns an object of `eav-data-attribute-option-interface` were `is_default` is to `true` 
 

### Expected result

1.  `is_default` return as a bool when present

### Actual result

1. `is_default` returns an object

### 2.2.6 SWAGGER Definition
```json
"eav-data-attribute-option-interface": {
			"type": "object",
			"description": "Created from:",
			"properties": {
				"label": {
					"type": "string",
					"description": "Option label"
				},
				"value": {
					"type": "string",
					"description": "Option value"
				},
				"sort_order": {
					"type": "integer",
					"description": "Option order"
				},
				"is_default": {
					"type": "boolean",
					"description": "Default"
				},
				"store_labels": {
					"type": "array",
					"description": "Option label for store scopes",
					"items": {
						"$ref": "#/definitions/eav-data-attribute-option-label-interface"
					}
				}
			},
			"required": [
				"label",
				"value"
			]
		},
```

### Sample of returned json 

`CatalogProductAttributeRepositoryV1Api.CatalogProductAttributeRepositoryV1GetListGet`

```json
"attribute_code": "country_of_manufacture",
   "options":[
     {
	"label": "United States",
         "value": "US",
	"is_default": [
		"1"
		]
    },
```

### Expected Return

```json

{
  "label": "United States",
  "value": "US",
  "is_default": "1"
},
```

**Comment by magento-engcom-team** [comment]
Hi @vetshopdeveloper. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me $VERSION instance` 

where `$VERSION` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (for example: 2.3-develop). 
For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@vetshopdeveloper do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by vetshopdeveloper** [comment]
@magento-engcom-team give me 2.2.6 instance

**Comment by magento-engcom-team** [comment]
Hi @vetshopdeveloper. Thank you for your request. I'm working on Magento 2.2.6 instance for you

**Comment by magento-engcom-team** [comment]
Hi @vetshopdeveloper, here is your [Magento instance](https://i-18525-2-2-6.engcom.dev.magento.com).
Admin access: [https://i-18525-2-2-6.engcom.dev.magento.com/admin](https://i-18525-2-2-6.engcom.dev.magento.com/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by vetshopdeveloper** [comment]
Confirmed on default instance provided

`https://i-18525-2-2-.engcom.dev.magento.com/rest/all/V1/products/attributes/country_of_manufacture`

swagger definition as per above instance attached as `swagger.clean.json.zip`
[swagger.clean.json.zip](https://github.com/magento/magento2/files/2468585/swagger.clean.json.zip)


```json
"is_default": {
                    "type": "boolean",
                    "description": "Default"
                },
````

return from above call attached as `response.zip`
[response.zip](https://github.com/magento/magento2/files/2468589/response.zip)


```json
{
      "label": "United States",
      "value": "US",
      "is_default": [
        "1"
      ]
    },
```

**Comment by vetshopdeveloper** [comment]
@magento-engcom-team ???

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-andrii. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [x] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [x] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [x] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [x] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [x] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [x] 6. Add label `Issue: Confirmed` once verification is complete. 

- [x] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by magento-engcom-team** [comment]
@engcom-backlog-andrii Thank you for verifying the issue. Based on the provided information internal tickets `MAGETWO-96632`, `MAGETWO-96633` were created

**Comment by magento-engcom-team** [comment]
Hi @hiren-wagento. Thank you for working on this issue. 
Looks like this issue is already verified and confirmed. But if your want to validate it one more time, please, go though the following instruction:
- [ ] 1. Add/Edit `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.

- [ ] 2. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 3. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 4. If the issue is not relevant or is not reproducible any more, feel free to close it.

**Comment by magento-engcom-team** [comment]
Hi @vetshopdeveloper. Thank you for your report. 
The issue has been fixed in magento/magento2#20913 by @hiren-wagento in 2.3-develop branch
Related commit(s):
 - [1e77ebe49c1e2a503a688eeeef850657c4661eeb](https://github.com/magento/magento2/commit/1e77ebe49c1e2a503a688eeeef850657c4661eeb)

The fix will be available with the upcoming 2.3.1 release.

**Comment by vetshopdeveloper** [comment]
Hi, Will the fix be back ported to the 2.2.x branch

**Comment by magento-engcom-team** [comment]
Hi @vetshopdeveloper. Thank you for your report. 
The issue has been fixed in magento/magento2#21164 by @mage2pratik in 2.2-develop branch
Related commit(s):
 - [11cafc3ce9b662ab0ea871c36761f59565343a37](https://github.com/magento/magento2/commit/11cafc3ce9b662ab0ea871c36761f59565343a37)
 - [c1f3f36b38b45fe94b54de7ec4785395a6c326b9](https://github.com/magento/magento2/commit/c1f3f36b38b45fe94b54de7ec4785395a6c326b9)
 - [7b8ec8085d29f84fadac67a4e17753a66f4e6ddf](https://github.com/magento/magento2/commit/7b8ec8085d29f84fadac67a4e17753a66f4e6ddf)

The fix will be available with the upcoming 2.2.9 release.
